tests/snapshots/assets__check_file@tablex.typ-80.snap

```diff
@@ -1475,7 +1480,10 @@ snapshot_kind: text
               align_default: auto,
             )
 
-            let width = measure(cell-box, styles).width // + 2*cell_inset // the box already considers inset
+            let width = measure(
+              cell-box,
+              styles,
+            ).width // + 2*cell_inset // the box already considers inset
```

Comment:
seems that in previous versions, when the body of the field access is a function call, we cannot correctly format the function call?